### DISCUSSION TITLE
Custom directory for snippets

### DISCUSSION THREAD

**Discussion by badasukerubin** [chakra-ui/chakra-ui#9057] [Q&A] [answered]
Currently, in v3, snippets get installed in the root `.src/**`, is there a way/option to install them in a custom directory say `.resources/js/**` for example?

My `.tsconfig `has paths defined as:
```json
"paths": {
    "@/*": ["./resources/js/*"]
},
```
in `compilerOptions` and I use `tsconfigPaths` as described in the docs

**Comment by segunadebayo** [comment, marked as answer]
Yes, please use `--outdir` flag
